### ISSUE TITLE
I get logged out of my accounts when I use the Regex101 website

### ISSUE THREAD

**Issue by ScraperMan2002** [firasdib/Regex101#2243]
## Description:
When I am using lots of pages of regex101 for my own sake, and when I am inputting large texts into the textboxes, I mysteriously get locked out of all my accounts.

### Reproduction steps
Open up lots of links, then input text sizes of 6510 characters or something. 
### Expected Outcome
You get logged out of all the accounts you're currently logged into.

### Browser
Chrome
### OS
MacOS

**Comment by working-name** [comment]
Hello @ScraperMan2002 

Not going to lie, it's pretty tough to understand how to try and reproduce this. Got some questions:

1. You're already logged in, and then you open up a bunch of previously saved links?
2. Are you maybe looking at a tab that was not refreshed after you logged in? (you will lose data if you didn't save the entry)
3. Are you sure it's the same browser opening the links as the one you're currently using and are logged in to regex101.com?
4. Do you get any issues loading all the pages?
5. Is cloudflare or regex101.com reacting to that behavior which can be defined as DDOSing? Any errors?

**Comment by working-name** [comment]
Closing for now, will re-open should you be able to help me try and reproduce it.

**Comment by ScraperMan2002** [comment]
Sorry for the late reply, I was busy with other work.

I don't even know how to reproduce it. It's just that after the day that I started using regex101, I would get logged out randomly by my browser without consent. It's super strange to me. And it always happens whenever I'm using Regex101, never when I'm doing anything else. I don't know what's going on, I don't even know if regex101 is the problem. I asked others and even ChatGPT, and they gave me no good answers

**Comment by working-name** [comment]
No worries.

Okay I'll try to do the same thing with a Chrome browser. Will report back.

Just out of curiosity: do regex101 tabs got to sleep (suspension, reduce RAM usage)? For me when tabs do that they reload the page upon reactivation. That might be an issue, coupled with loss of cookies - either an add-on doing that, or you, manually?

**Comment by working-name** [comment]
Hi @ScraperMan2002 

Here's Chrome 1 month later. It didn't make the Regex101 tabs go to sleep although I enabled that option. I did not get logged out, nor did I lose any of the content in the 7 tabs I opened back then. Each tab has 4,000 to 7,000 characters in the test string.

https://github.com/firasdib/Regex101/assets/16505052/c2b6287d-6bef-4107-861a-69f0eae2a758


That being said:
- the mac mini did not reboot
- chrome was not quit
- I did not clear any cookies

So I'm not sure how else to try to reproduce this. Any other ideas?
